### PR TITLE
Adding Support for Django 3.0

### PR DESCRIPTION
**Pull request by samatix**
Following the issue with six that has been removed in Django 3.0, I have amended the pagination file to use the text_type function from six directly and added tests in tox on py37 and django 3.0

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -5,18 +5,22 @@ envlist =
        py27-django1.{10,11}-drf3.{5,6,7,8}
        {py34,py35,py36}-django2.0-drf3.{7,8,9}
        {py35,py36}-django2.1-drf3.{8,9}
+       {py35,py36}-django2.1-drf3.{8,9}
+       {py37}-django3.0-drf3.{8,9}
```
Comment (CarlosBorroto) [inline tox.ini]: It looks like your build is failing cause drf just added Django 3.0 support in 3.11. Can you try adding `drf3.11: djangorestframework>=3.11,<3.12` below and changing this to `{py37}-django3.0-drf3.{11}`.

Looking forward to 3.0 compatibility.

Comment (samatix) [reply tox.ini]: Thanks Carlos

**Comment by Simonefardella** [comment]
Please Release the new package version!

**Comment by ramonsaraiva** [comment]
Maybe you could also include py 3.8 in travis? Also, @CarlosBorroto could you prioritize releasing this?

**Comment by CarlosBorroto** [comment]
@ramonsaraiva I don't have control over this repo. I'm also very interested in this PR and that is why I took a look into the failed tests.

**Comment by Simonefardella** [comment]
@izimobil do you have any schedule about the new release? do you need some maintainers for this repo? in any case i'm available to do that.

**Comment by izimobil** [comment]
I'm not a fan of the added six dependency... Can someone update this PR with something like this instead ?
```
try:
    from django.utils import six
    text_type = six.text_type
except ImportError:
    text_type = str
```

**Comment by izimobil** [comment]
Also two things are missing IMO:
* Add python 3.8 in travis.yml and tox.ini
* Add a note in the docs regarding Django/DRF/DRF-datatables version compatibility

I just don't have enough time to do it myself right now.

Once we have this done, I'll happily release a new version to pypi.
Thanks.

**Comment by samatix** [comment]
Done @izimobil

**Comment by izimobil** [comment]
Thanks !

**Comment by ramonsaraiva** [comment]
Thanks guys!

**Comment by TauPan** [comment]
Any reason why Django 2.2 (the current LTS release) is not in tox.ini? My current target platform is python 3.6, Django 2.2 and django rest framework >= 3.9 (not really relevant if newer, since I'm packaging that myself).

Would you mind if I prepare another pull request for that @izimobil?

**Comment by izimobil** [comment]
@TauPan no particular reason, sure go ahead ! Thx.

**Pull request merged**